src/classes/CON_ContactMerge_CTRL.cls

```diff
@@ -274,6 +274,16 @@ public with sharing class CON_ContactMerge_CTRL {
             Schema.SobjectField f = contactFields.get(fieldName);
             Schema.DescribeFieldResult fResult = f.getDescribe();
             
+            // Ignore fields that have an identical API name as an npsp prefixed field
+            // This is due to a bug in dynamic SOQL, quich causes querying for two real fields, 
+            // e.g. npsp__foo__c and foo__c, to throw a duplicate field selected error.
+            // Only select the NPSP namespaced field and ignore the same field without namespace
+            if (fieldName.countMatches('__') == 1 && 
+                contactFields.containsKey(UTIL_Namespace.StrTokenNSPrefix(fieldName))) {
+                
+                continue;
+            }
+
```

Comment:
in unmanaged code, won't this have the affect of ignoring all of our npsp fields?